transfer/box.hpp

```diff
@@ -0,0 +1,82 @@
+#ifndef MFEM_L2P_BOX_HPP
+#define MFEM_L2P_BOX_HPP 
+
+#include "../fem/fem.hpp"
+
+
+namespace mfem {
+
+	void MaxCol(const DenseMatrix &mat, Vector &vec, bool include_vec_elements);
+	void MinCol(const DenseMatrix &mat, Vector &vec, bool include_vec_elements);
+
+	class Box {
```

Comment:
Why is this functionality in MFEM and not in Moonolith?

Reply:
Removed the class and used moonolith bounding volumes instead